core_values.py

```diff
@@ -0,0 +1,18 @@
+class CoreValues:
+    def __init__(self):
+        self.values = {}
```

Comment:
Could add type hints to the dictionary of values so that it's more strongly typed like so:
```suggestion
        self.values: Dict[str,float] = {}
```